terminus/generators/opendrive_id_mapper.py

```diff
@@ -0,0 +1,11 @@
+from city_visitor import CityVisitor
+
+
+class OpenDriveIdMapper(CityVisitor):
```

Comment:
Since ids are just incremental I think we can have this in the generator itself.